Description: Fix typo that caused the Options dialog to appear blank
Bug: http://code.google.com/p/google-gadgets-for-linux/issues/detail?id=350
Bug-Ubuntu: https://bugs.launchpad.net/googlegadgets/+bug/507542
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=566168
Author: Michael Terry <michael.terry@canonical.com>

--- a/gadgets/digital_alarm_clock/options.js
+++ b/gadgets/digital_alarm_clock/options.js
@@ -24,7 +24,7 @@
 /**
  * Method called when options are opened
  */
-function onOpen() {
+function _onOpen() {
   oString1.innerText = ALARM_SET;
   oString2.innerText = DEFAULT_ALARM;
   oString3.innerText = USE_FILE;
